packages/widgets/src/utils.js

```diff
@@ -23,11 +27,33 @@ export function getWidgetIdFromBlock( block ) {
  * @return {Block} The updated block.
  */
 export function addWidgetIdToBlock( block, widgetId ) {
+	block.attributes.__internalWidgetId = widgetId;
```

Comment:
What is this change for? Redux and `@wordpress/data` often rely on object reference equality (`foo === bar`) to detect if something has changed, so not sure that it's a good idea to **not** create a new `attributes` object here.

Reply:
No good reason, crossed brain wires as I was thinking about updating an attribute now guaranteed to exist -- you're of course right. Fixed, thanks!